src/compounds/Document/Document.jsx

```diff
@@ -97,16 +94,16 @@ const Document = ({ document, addClass }) => {
 Document.propTypes = {
   addClass: PropTypes.string,
   document: PropTypes.shape({
-    adPO: allowNull(PropTypes.string.isRequired),
+    adPO: PropTypes.string,
```

Comment:
I separted the sow and po documents in the webstore so we don't need to account for null values here anymore. 

the function itself could be useful in the future though, so I left it.

Reply:
the allowNull function is already being used in several other places i believe

Reply:
ahhh ok. cool.